### PR TITLE
fix bug in issue #828

### PR DESCRIPTION
**Pull request by sfzeng**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
```

### CODE REVIEW

File: s3/pkg/service/object.go [hunk 1]
```diff
@@ -513,7 +513,8 @@ func (s *s3Service) CopyObject(ctx context.Context, in *pb.CopyObjectRequest, ou
 		log.Errorln("failed to create storage. err:", err)
 		return err
 	}
-
+	log.Debugf("***********************ctx:%+v\n", ctx)
```
Comment (kumarashit) [inline s3/pkg/service/object.go]: Are these logs required?

File: s3/pkg/service/object.go [hunk 2]
```diff
@@ -531,6 +532,7 @@ func (s *s3Service) CopyObject(ctx context.Context, in *pb.CopyObjectRequest, ou
 		targetObject.ObjectId = oldObj.ObjectId
 	}
 	ctx = context.WithValue(ctx, dscommon.CONTEXT_KEY_SIZE, srcObject.Size)
+	log.Info("put object ....")
```
Comment (kumarashit) [inline s3/pkg/service/object.go]: Same here

File: s3/pkg/service/object.go (hunk 1 above)
Comment (sfzeng) [reply s3/pkg/service/object.go]: It's for debug

File: s3/pkg/service/object.go (hunk 2 above)
Comment (sfzeng) [reply s3/pkg/service/object.go]: Download and upload speed of some backends are slow, so, add log here for trace.

**Comment by wisererik** [comment]
@sfzeng please add test report about this pr

**Comment by sfzeng** [comment]
fix828_dashboard
fix828_downloaded_file

File: s3/pkg/service/object.go (hunk 2 above)
Comment (sfzeng) [reply s3/pkg/service/object.go]: Removed.

**Pull request merged**